resources/testharness.js

```diff
@@ -1934,9 +2032,33 @@ policies and contribution forms [3].
     };
 
     Tests.prototype.timeout = function() {
+        var cleaningTest = null;
+
         if (this.status.status === null) {
-            this.status.status = this.status.TIMEOUT;
+            forEach(this.tests,
+                    function(test) {
+                        if (test.phase === test.phases.CLEANING) {
+                            cleaningTest = test;
+                        }
+
+                        test.phase = test.phases.COMPLETE;
+                    });
+
+            // Timeouts that occur while a sub-test is in the "cleanup" phase
```

Comment:
s/sub-test/test/ here and below, other comments in this file just call them tests.

Reply:
You got it. (This was an attempt to be a little more precise with the terminology, but it's not helping anyone to be inconsistent.)